{"uuid": "519f949f-70d5-4515-b71a-ce71c596ba40", "counter": 38, "stdout": "fatal: [server1]: FAILED! => {\"changed\": true, \"cmd\": [\"docker\", \"run\", \"-d\", \"--name\", \"3x-ui-1bakvivas\", \"-p\", \"65432:65432\", \"3x-ui-1bakvivas:latest\"], \"delta\": \"0:00:00.259123\", \"end\": \"2024-12-03 16:16:40.758941\", \"msg\": \"non-zero return code\", \"rc\": 125, \"start\": \"2024-12-03 16:16:40.499818\", \"stderr\": \"docker: Error response from daemon: Conflict. The container name \\\"/3x-ui-1bakvivas\\\" is already in use by container \\\"f2b592ef5690c02e48bce8f5bddcc1006e899748d4ca329552356dc2cccd042f\\\". You have to remove (or rename) that container to be able to reuse that name.\\nSee 'docker run --help'.\", \"stderr_lines\": [\"docker: Error response from daemon: Conflict. The container name \\\"/3x-ui-1bakvivas\\\" is already in use by container \\\"f2b592ef5690c02e48bce8f5bddcc1006e899748d4ca329552356dc2cccd042f\\\". You have to remove (or rename) that container to be able to reuse that name.\", \"See 'docker run --help'.\"], \"stdout\": \"\", \"stdout_lines\": []}", "start_line": 43, "end_line": 44, "runner_ident": "e5a0773b-10d0-4ddd-b7b9-91e13bc8d948", "event": "runner_on_failed", "pid": 60507, "created": "2024-12-03T16:16:40.956105+00:00", "parent_uuid": "985f4199-2370-ba37-3e22-000000000012", "event_data": {"playbook": "playbook.yml", "playbook_uuid": "710115aa-fe3e-4c28-b7ee-e512ca4b5199", "play": "start docker", "play_uuid": "985f4199-2370-ba37-3e22-000000000002", "play_pattern": "3x_ui_server", "task": "Run VPN container", "task_uuid": "985f4199-2370-ba37-3e22-000000000012", "task_action": "ansible.builtin.command", "resolved_action": "ansible.builtin.command", "task_args": "", "task_path": "/home/bakvivas/Documents/Projects/ansible-role-3x-ui/roles/3x_ui_container/tasks/main.yml:29", "role": "3x_ui_container", "host": "server1", "remote_addr": "server1", "res": {"changed": true, "stdout": "", "stderr": "docker: Error response from daemon: Conflict. The container name \"/3x-ui-1bakvivas\" is already in use by container \"f2b592ef5690c02e48bce8f5bddcc1006e899748d4ca329552356dc2cccd042f\". You have to remove (or rename) that container to be able to reuse that name.\nSee 'docker run --help'.", "rc": 125, "cmd": ["docker", "run", "-d", "--name", "3x-ui-1bakvivas", "-p", "65432:65432", "3x-ui-1bakvivas:latest"], "start": "2024-12-03 16:16:40.499818", "end": "2024-12-03 16:16:40.758941", "delta": "0:00:00.259123", "msg": "non-zero return code", "invocation": {"module_args": {"_raw_params": "docker run -d --name 3x-ui-1bakvivas -p 65432:65432 3x-ui-1bakvivas:latest", "_uses_shell": false, "expand_argument_vars": true, "stdin_add_newline": true, "strip_empty_ends": true, "argv": null, "chdir": null, "executable": null, "creates": null, "removes": null, "stdin": null}}, "stdout_lines": [], "stderr_lines": ["docker: Error response from daemon: Conflict. The container name \"/3x-ui-1bakvivas\" is already in use by container \"f2b592ef5690c02e48bce8f5bddcc1006e899748d4ca329552356dc2cccd042f\". You have to remove (or rename) that container to be able to reuse that name.", "See 'docker run --help'."], "_ansible_no_log": false}, "start": "2024-12-03T16:16:38.358592+00:00", "end": "2024-12-03T16:16:40.955924+00:00", "duration": 2.597332, "ignore_errors": null, "event_loop": null, "uuid": "519f949f-70d5-4515-b71a-ce71c596ba40"}}